private/data/i18n/en.json

```diff
@@ -290,19 +539,59 @@
         "text": "Create"
       },
       "social": {
+        "socialTitle": "Social",
         "facebook": "Facebook",
         "google": "Google",
         "github": "GitHub",
+        "instagram": "Instagram",
         "meetup": "Meetup",
+        "ok": "Ok",
         "twitter": "Twitter",
+        "vk": "VK",
         "weibo": "Weibo"
       },
+      "search": {
+        "clearSearch": "Clear",
+        "noProductsFound": "No products found.",
+        "noOrdersFound": "No orders found.",
+        "noAccountsFound": "No accounts found.",
+        "searchInputLabel": "Search",
+        "suggestedTags": "Suggested tags",
+        "searchTypeProducts": "Products",
+        "searchTypeAccounts": "Accounts",
+        "searchTypeOrders": "Orders",
+        "accountSearchResults": {
+          "accountId": "Account ID",
+          "shopId": "Shop ID",
+          "firstName": "First name",
+          "lastName": "Last name",
+          "phone": "Phone",
+          "emails": "Email"
+        },
+        "orderSearchResults": {
+          "orderId": "Order ID",
+          "shopId": "Shop ID",
+          "shippingAddress": "Address",
+          "shippingCity": "City",
+          "shippingRegion": "Region",
+          "shippingCountry": "Country",
+          "shippingPhone": "Phone",
+          "shippingName": "Name",
+          "shippingStatus": "Status",
+          "billingAddress": "Address",
+          "billingCity": "City",
+          "billingRegion": "Region",
+          "billingCountry": "Country",
+          "billingPhone": "Phone",
+          "billingStatus": "Status",
+          "userEmails": "Email",
+          "orderDate": "Date",
+          "orderTotal": "Total"
+        }
+      },
       "accountsUI": {
-        "yourOrders": "Your orders",
```

Comment:
These were added recently

Reply:
This should have merged master into release-1.50, shouldn't have deleted anything new, just merged some old stuff back in, and potentially overwrote updates text with old text (mostly sentence case which I went through and fixed)